js/src/setup-mc/setup-stepper/setup-accounts/wordpressdotcom-account/card-content.js

```diff
@@ -30,7 +31,8 @@ const CardContent = () => {
 	}
 
 	if ( jetpack.active === 'yes' ) {
-		return <TitleButtonLayout title={ jetpack.email } />;
+		const title = getConnectedJetpackInfo( jetpack );
+		return <TitleButtonLayout title={ title } />;
```

Comment:
💅 😄  very small nit, we can just squeeze it like this, similar to `DisconnectAccounts`.

```suggestion
		return (
			<TitleButtonLayout title={ getConnectedJetpackInfo( jetpack ) } />
		);
```